src/test/java/bio/terra/service/snapshotbuilder/query/LiteralTest.java

```diff
@@ -38,7 +38,7 @@ void renderInt(SqlRenderContext context) {
   @ArgumentsSource(QueryTestUtils.Contexts.class)
   void renderBoolean(SqlRenderContext context) {
     var literal = new Literal(true);
-    assertThat(literal.renderSQL(context), is("true"));
+    assertThat(literal.renderSQL(context), is(context.getPlatform().choose("true", "1")));
```

Comment:
Did you include a test where `literal = new Literal(false);`?

Reply:
No, but I can add it